### DISCUSSION TITLE
Search contents among versions

### DISCUSSION THREAD

**Discussion by ncmadhan1989** [microsoft-search/pnp-modern-search#3248] [Q&A] [answered]
Did Pnp Modern search webpart support search content from the file among versions?

If not please suggest some workarounds

**Comment by kasperbolarsen** [comment, marked as answer]
Please see https://techcommunity.microsoft.com/t5/sharepoint/search-content-from-version-history-versions/m-p/815920
Only the current version is available in Search.

The repository is about PnP Modern Search. I suggest you ask in the Tech forums about any worksrounds.

**Comment by wobba** [comment]
@ncmadhan1989 Can you share the scenario where you want to search in versions, and how would you like to visualize this? There is a preview premium feature for eDiscovery which allows version searching (https://learn.microsoft.com/en-us/purview/ediscovery-historical-versions) and would like to know how you might want to use such a feature outside of eDiscovery.
